### PR TITLE
Vulkan: Implement support for post-processing

### PR DESCRIPTION
**Pull request by stenzek**
Nothing too fancy or any new features here, just bringing the Vulkan backend up to feature parity with OpenGL.

### CODE REVIEW

**Comment by JMC47** [comment]
Maybe you can finish the Post Processing rewrite sometime to get all backends up to parity too.  While this is nice, it just reminds me of unfinished shenanigans.

File: Source/Core/VideoBackends/Vulkan/Renderer.cpp [hunk 1]
```diff
@@ -1490,31 +1484,14 @@ bool Renderer::CompileShaders()
...
   std::string source;
 
   source = header + CLEAR_FRAGMENT_SHADER_SOURCE;
   m_clear_fragment_shader = Util::CompileAndCreateFragmentShader(source);
-  source = header + BLIT_FRAGMENT_SHADER_SOURCE;
-  m_blit_fragment_shader = Util::CompileAndCreateFragmentShader(source);
 
-  if (m_clear_fragment_shader == VK_NULL_HANDLE || m_blit_fragment_shader == VK_NULL_HANDLE)
-  {
+  if (m_clear_fragment_shader == VK_NULL_HANDLE)
```
Comment (iwubcode) [inline Source/Core/VideoBackends/Vulkan/Renderer.cpp]: return m_clear_fragment_shader != VK_NULL_HANDLE;   ?

**Comment by iwubcode** [comment]
Code looks good.  Also, I tested this with a couple games and a shader or two.  It works!

**Comment by JMC47** [comment]
LGTM.  NOTE: I don't review the code, so, this should be just from a usage angle.

**Pull request merged**